_pages/welcome-to-18f/classes/gsa-tools-equipment-and-transit.md

```diff
@@ -136,7 +136,7 @@ If you&rsquo;re new to GSA, it&rsquo;ll take a few weeks for IT to set up your a
 3. After you activate your account, you can set up your login ID and password.
 4. Then, go back to the welcome email and click the second link to log in.
 
-Once your account is setup, be sure to log in right away as that initial access expires within a few days. If you have trouble logging in, call the Employee Express help desk at 1-888-353-9450 or email [eexhelp@opm.gov](mailto:eexhelp@opm.gov). They&rsquo;ll ask you for your name, date of birth, agency (OCSIT/18F), work email address, and request type (for example, `my SSN is not recognized` or `I need my login and password reset`).
+Once your account is setup, be sure to log in right away as that initial access expires within a few days. If you have trouble logging in or need to contact the Employee Express Help Desk, send an email to [eexhelp@opm.gov](mailto:eexhelp@opm.gov) from your GSA email address. In the email, include your name, work phone number and a description of your request(for example, `my SSN is not recognized` or `I need my login and password reset`).
```

Comment:
Thanks for this update! There's a missing space after "a description of your request".